kpatch/kpatch

```diff
@@ -165,7 +165,27 @@ load_module () {
 	fi
 
 	echo "loading patch module: $module"
-	insmod "$module"
+	local i=0
+	while true; do
+		out=$(insmod "$module" 2>&1)
+		[[ -z $out ]] && break
+		echo $out 1>&2
+		[[ ! $out =~ "Device or resource busy" ]] &&
```

Comment:
@joe-lawrence That's an interesting question. I looked at the insmod sources from kmod and old module-init-tools sources (superceded by kmod) and they both have always used `strerror` to print the message, *not* `strerror_l`,  which would print the locale-dependent/translated version of the error message. So I think the message will always be in English.

(For the curious, the translated glibc error messages used by `strerror_l` and others can be found here: https://sourceware.org/git/?p=glibc.git;a=tree;f=po;h=b6fb91b5b5b9730f24a0960bf0b0db766fc54c17;hb=HEAD)